apps/chef_test/src/chef_test_suite_helper.erl

```diff
@@ -106,8 +127,90 @@ make_az_id(Prefix) ->
 actor_id() ->
     make_az_id(<<"ffff">>).
 
+make_orgs() ->
+    make_org(),
+    make_other_org(),
+    OrgConfig = org_config(),
+    confirm_org_setup(OrgConfig),
+    OrgConfig.
+
+make_org() ->
+    Org = chef_object:new_record(oc_chef_organization, nil, ?ORG_AUTHZ_ID,
+                                 {[{<<"name">>, ?ORG_NAME}, {<<"full_name">>, ?ORG_NAME}]}),
+    ok = chef_db:create(Org, context(), ?ORG_AUTHZ_ID).
+
+make_other_org() ->
+    Org = chef_object:new_record(oc_chef_organization, nil, ?OTHER_ORG_AUTHZ_ID,
+                                 {[{<<"name">>, ?OTHER_ORG_NAME}, {<<"full_name">>, ?OTHER_ORG_NAME}]}),
+    ok = chef_db:create(Org, context(), ?OTHER_ORG_AUTHZ_ID).
+
+org_config() ->
+    OrgId = chef_db:fetch_org_id(context(), ?ORG_NAME),
+    OtherOrgID = chef_db:fetch_org_id(context(), ?OTHER_ORG_NAME),
+    ct:pal("Other org: ~p", [OtherOrgID]),
+    [{org_id, OrgId}, {other_org_id, OtherOrgID}].
+
+confirm_org_setup(Config) ->
+    ExpectedOrgId = proplists:get_value(org_id, Config),
+    ExpectedOtherOrgId = proplists:get_value(other_org_id, Config),
+
+    ct:pal("ORG: ~p~n", [ExpectedOrgId]),
+    ct:pal("OTHER ORG: ~p~n", [ExpectedOtherOrgId]),
+
+    ActualMainOrg = chef_db:fetch_org_id(context(), ?ORG_NAME),
+    ActualOtherOrg = chef_db:fetch_org_id(context(), ?OTHER_ORG_NAME),
+    ?assertEqual(ExpectedOrgId, ActualMainOrg),
+    ?assertEqual(ExpectedOtherOrgId, ActualOtherOrg).
+
+
+org_name() ->
+    ?ORG_NAME.
+
+other_org_name() ->
+    ?OTHER_ORG_NAME.
+
+context() ->
+    chef_db:make_context(<<"AB">>).
+
 the_org_id() ->
     make_id(<<"aa1">>).
 
 other_org_id() ->
     make_id(<<"bb2">>).
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Common database queries
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+create_record(Record) ->
+    Query = chef_object:create_query(Record),
```

Comment:
For the chef_db case, we definitely don't want to use chef_db because then your test setup/teardown will rely on the same code as you're testing. When testing other modules "above" the chef_db level, you could argue either way. Going the non-chef_db route means you skip callbacks like set_created, set_updated, etc. so it gives you a bit more control over how you setup your fixture data. On the other hand, I'm not 100% sure this is something you would need, but of course if you don't need it, you could just use the chef_db stuff. Thoughts?